proposals/0018-spirv-resource-representation.md

```diff
@@ -66,7 +67,7 @@ type. For now, the counter variable will not be handled. We will create a target
 type `spirv.VulkanBuffer` to represent a storage or uniform buffer:
 
 ```
-target("spirv.VulkanBuffer", ElementType, StorageClass, IsWriteable, IsROV)
```

Comment:
You removed `IsROV` but the lines below still mentions it.

Reply:
Thanks. I missed that.